pages/common/gnmic.md

```diff
@@ -0,0 +1,24 @@
+# gnmic
+
+> A gNMI CLI client that provides full support for Capabilities, Get, Set and Subscribe RPCs with collector capabilities.
+> More information: <https://gnmic.kmrd.dev>.
+
+- Request capabilities (gnmic capabilities):
+
+`gnmic --address {{ip:port}} --insecure capabilities`
+
+- Username and password can be given to any command:
+
+`gnmic --address {{ip:port}} --username {{username}} --password {{password}} --insecure capabilities`
+
+- Get request (gnmic get):
+
+`gnmic -a {{ip:port}} --insecure get --path {{path}}`
+
+- Set request (gnmic set):
+
+`gnmic -a {{ip:port}} --insecure set --update-path {{path}} --update-value {{value}}`
+
+- Subscribe request (gnmic subscribe):
+
+`gnmic -a {{ip:port}} --insecure subscribe --path {{path}}`
```

Comment:
Going off of the [gNMI reference section 3.5](https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-specification.md#35-subscribing-to-telemetry-updates):

```suggestion
- Subscribe to target state updates under the subtree {{path}}:

`gnmic -a {{ip:port}} --insecure subscribe --path {{path}}`
```